src/app/auth/register/register.component.html

```diff
@@ -21,24 +20,23 @@ <h2>Register</h2>
             <div *ngIf="registerForm.controls.email.dirty && (registerForm.get('email')?.status === 'INVALID')">
               <p class="help is-danger">Must be a valid Email.</p>
             </div>
-            <div *ngIf="registerForm.controls.email.dirty && (!uniqueE)">
+            <div *ngIf="registerForm.controls.email.dirty && (!uniqueEmailFlag)">
```

Comment:
Use angular forms methods to get dirty status. Use uniqueness error handling instead of a flag